src/DynamoCoreWpf/UI/GuidedTour/Survey.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls.Primitives;
+
+namespace Dynamo.Wpf.UI.GuidedTour
+{
+    public class Survey : Step
+    {
+        public Survey(HostInfo host, double width, double height)
+            :base(host, width, height)
+        {
+
```

Comment:
Same here

Reply:
@QilongTang added a comment in this line